### ISSUE TITLE
Flutter crash on Android

### ISSUE THREAD

**Issue by SergeiEensalu** [Sh1d0w/multi_image_picker#508]
**Bug Report**

`multi_image_picker` version: ^4.6.9

Is it happening on Android: Yes

Is it happening on iOS: No

```


[✓] Flutter (Channel master, 1.21.0-6.0.pre.17, on Mac OS X 10.15.6 19G73, locale en-EE)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✗] Xcode - develop for iOS and macOS
    ✗ Xcode installation is incomplete; a full installation is necessary for iOS development.
      Download at: https://developer.apple.com/xcode/download/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install:
        sudo gem install cocoapods
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Ultimate Edition (version 2019.3.3)
[✓] Connected device (3 available)

! Doctor found issues in 1 category. 

```
```
 List<Asset> pickedImagesList = await MultiImagePicker.pickImages(maxImages: 25);
```

```
E/MethodChannel#multi_image_picker( 6241): Failed to handle method call
E/MethodChannel#multi_image_picker( 6241): java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter activity
E/MethodChannel#multi_image_picker( 6241): 	at com.sangcomz.fishbun.FishBun$Companion.with(Unknown Source:2)
E/MethodChannel#multi_image_picker( 6241): 	at com.sangcomz.fishbun.FishBun.with(Unknown Source:2)
E/MethodChannel#multi_image_picker( 6241): 	at com.vitanov.multiimagepicker.MultiImagePickerPlugin.presentPicker(MultiImagePickerPlugin.java:535)
E/MethodChannel#multi_image_picker( 6241): 	at com.vitanov.multiimagepicker.MultiImagePickerPlugin.onMethodCall(MultiImagePickerPlugin.java:259)
E/MethodChannel#multi_image_picker( 6241): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:230)
E/MethodChannel#multi_image_picker( 6241): 	at io.flutter.embedding.engine.dart.DartMessenger.handleMessageFromDart(DartMessenger.java:85)
E/MethodChannel#multi_image_picker( 6241): 	at io.flutter.embedding.engine.FlutterJNI.handlePlatformMessage(FlutterJNI.java:683)
E/MethodChannel#multi_image_picker( 6241): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/MethodChannel#multi_image_picker( 6241): 	at android.os.MessageQueue.next(MessageQueue.java:326)
E/MethodChannel#multi_image_picker( 6241): 	at android.os.Looper.loop(Looper.java:160)
E/MethodChannel#multi_image_picker( 6241): 	at android.app.ActivityThread.main(ActivityThread.java:6669)
E/MethodChannel#multi_image_picker( 6241): 	at java.lang.reflect.Method.invoke(Native Method)
E/MethodChannel#multi_image_picker( 6241): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
E/MethodChannel#multi_image_picker( 6241): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
I/flutter ( 6241): Oopss...PlatformException(error, Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter activity, null)
```

**Describe the bug**

This https://github.com/Sh1d0w/multi_image_picker/issues/374 doesn't help

**Comment by vishalsaini9** [comment]
Getting the same issue 
This #374 doesn't help

**Comment by Firman95** [comment]
this pull request solves the problem, I tried it, but haven't merged yet https://github.com/Sh1d0w/multi_image_picker/pull/506

**Comment by SergeiEensalu** [comment]
@Firman95 When will your decision be implemented approximately? Thank you :)

**Comment by konewltd** [comment]
same problem, please fix as soon as possible!

**Comment by Sh1d0w** [comment]
> this pull request solves the problem, I tried it, but haven't merged yet #506

This is now released in version 4.7.10.
